lib/ansible/modules/network/apconos/apconos_cert.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+#
+# Copyright (C) 2018 Apcon.
+#
+# GNU General Public License v3.0+
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+# Module to execute apconos Commands on Apcon Switches.
+# Apcon Networking
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: apconos_cert
+version_added: "2.9.0"
```

Comment:
It could also be 2.9 as no new modules are added on minor releases.

Reply:
I'll fix it!
Thanks!